### ISSUE TITLE
Redundant quoting of atoms

### ISSUE THREAD

**Issue by triska** [aarroyoc/advent-of-code-2020#5]
One very small cosmetic issue I noticed: There is a tendency to use redundant single quotes for atoms. None of the atoms `.`, `#`, `*` etc. need to be quoted.

For example, it is perfectly valid to write a term `V = vertex(X,Y,.)` or `T = f(#)`.

Single quotes have to be used for atoms like `'one atom'`.

**Comment by aarroyoc** [comment]
Yeah, it's true. Sometimes I wanted to be extra sure :), I'll try to fix them later
